Model/Api/Debug.php

```diff
@@ -198,11 +197,18 @@ public function universalDebug($type)
             );
         }
 
-        // Throw debg not implemented exception for now untill it is implemented.
-        throw new BoltException(
-            __('Webhook of type Debug has not been implemented'),
-            null,
-            BoltErrorResponse::ERR_SERVICE
-        );
+        $result = $this->debugInfoFactory->create();
+        switch (isset($data['type']) ? $data['type'] : ''){
+            case 'log':
+                $result->setLogs($this->logRetriever->getLogOfType(isset($data['log_type']) ? $data['log_type'] : null));
```

Comment:
What happens if null is passed to `getLogOfType`? Does it throw an exception of some kind or just return an empty result?

Reply:
Returns default logs.